LICENSE.md

```diff
@@ -1,9 +1,25 @@
-The MIT License (MIT)
+Copyright © 2017 Addepar, Inc. All Rights Reserved.
```

Comment:
This license is the same as the old one:

https://github.com/Addepar/ember-table/blob/addepar/LICENSE.md

Only the year is changed.